src/baseballGame/baseballGame.js

```diff
@@ -0,0 +1,58 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MSG, ERROR_MSG } from "../Messages";
+import { getComputerNum, getUserNum } from "./getValue";
+
+// 3. 컴퓨터와 유저의 숫자 비교
+export async function compareNum() {
+  const computer_num = getComputerNum();
+
+  let ball = 0;
+  let strike = 0;
+  while (strike !== 3) {
+    let user_num = await getUserNum();
+    user_num = user_num.split("").map(Number);
+    ball = 0;
+    strike = 0;
+
+    for (let i = 0; i < 3; i++) {
+      if (computer_num[i] === user_num[i]) strike++;
+      else if (user_num.includes(computer_num[i])) ball++;
+    }
+
+    getHint(ball, strike);
+  }
+
+  MissionUtils.Console.print(GAME_MSG.END);
+  await reStartOrNot();
+}
+
+// 4. 힌트 출력
+function getHint(ball, strike) {
+  if (ball === 0 && strike === 0) {
+    MissionUtils.Console.print(GAME_MSG.NOTHING);
+  } else if (ball > 0 && strike === 0) {
+    MissionUtils.Console.print(ball + GAME_MSG.BALL);
+  } else if (ball === 0 && strike > 0) {
+    MissionUtils.Console.print(strike + GAME_MSG.STRIKE);
+  } else {
+    MissionUtils.Console.print(
+      ball + GAME_MSG.BALL + " " + strike + GAME_MSG.STRIKE
+    );
+  }
+}
+
+// 6. 게임 재시작/종료 선택
+async function reStartOrNot() {
+  let answer = await MissionUtils.Console.readLineAsync(GAME_MSG.RESTART);
```

Comment:
`answer`  변수는 한 번 입력받으면 다시 변경되는 변수가 아니라서 const로 선언하는게 더 좋을 것 같습니다😀
```suggestion
  const answer = await MissionUtils.Console.readLineAsync(GAME_MSG.RESTART);
```